lua/astrocommunity/pack/python/init.lua

```diff
@@ -16,9 +16,7 @@ return {
   },
   {
     "jay-babu/mason-null-ls.nvim",
-    opts = function(_, opts)
-      opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, { "isort", "black" })
-    end,
+    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, { "black" }) end,
```

Comment:
```suggestion
    opts = function(_, opts) opts.ensure_installed = utils.list_insert_unique(opts.ensure_installed, "black") end,
```

We can simplify this to not a table anymore

Reply:
I went ahead and modified the commit to get this merged in :)